src/connector/src/sink/kinesis.rs

```diff
@@ -249,12 +236,36 @@ impl KinesisSinkWriter {
             &self.sink_from_name,
         );
 
-        crate::impl_load_stream_write_record!(dbz_stream, self.put_record);
+        #[for_await]
```

Comment:
We may also support dbz format in this PR?

Reply:
I'd still prefer to keep them separated as the code movement from `utils` to `formatter::debezium_json` would be quite long and could blur the focus of this PR.